readme.md

```diff
@@ -124,6 +124,11 @@ They try to be very welcoming for 1st time contributors.
 
 - [atom-icons](https://github.com/HackeSta/atom-icons)
 
+### TypeScript
+
+([all TypeScript projects with hacktoberfest issues](https://github.com/search?utf8=%E2%9C%93&q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+language%3ATypescript&type=Issues&ref=advsearch&l=Typescript))
+
+
```

Comment:
Nitpick: Please follow [MD-0022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022) with default values: Just one empty line around each heading. This is done by the other headings in this markdown file, too.